assets/cms/components/gallery/ImageDetail.vue

```diff
@@ -0,0 +1,93 @@
+<template functional>
+    <div class="ccm-gallery-image-details">
+        <div class="image-preview text-center">
+            <img :src="props.image.imageUrl" />
+            <button class="btn btn-secondary" @click="listeners.delete">Remove from Gallery</button>
+        </div>
+        <div class="image-details">
+            <section>
+                <strong>Custom Attributes</strong>
+
+                <p class="image-title">{{props.image.title}}</p>
+                <p class="image-description">{{props.image.description}}</p>
+
+                <div class="mb-4 text-right">
+                    <a :href="props.image.detailUrl" target="_blank" class="btn btn-secondary">Edit Attributes</a>
+                </div>
+            </section>
+            <section v-if="!props.image.displayChoices.lenght">
```

Comment:
Looks like a typo, `length` instead of `lenght`

Reply:
just fixed this typo, thank you @KorvinSzanto :)